generators/client/templates/react/src/main/webapp/app/shared/util/url-utils.ts.ejs

```diff
@@ -16,6 +16,40 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -%>
+export const parse = (header: string): any => {
```

Comment:
can we use a lib instead doing it ourself, something like https://github.com/CassetteRocks/react-infinite-scroller can you please explore